governance-exercises/Interpretation Exercise Response.md

```diff
@@ -0,0 +1,46 @@
+**Do you think there is any ambiguity in these clauses? If so, where, and how would you rewrite the clauses to remove the ambiguity?**
+
+Yes, recent events over the EDSR made it very clear ambiguity exists. Various community members were unsure on if this clause would allow for such a vote. One reasonable interpretation is that this power was meant to be used to fix obvious errors or weird edge cases that break things, but the author clarified that it could be used to deal with broader protocol issues(such as low DSR utilization and declining DAI supply). 
+
+Given the author's intent, I would suggest  the following edits be included in a future Scopes update to clarify that the power is intended to be this broad. To be clear, this does not necessarily mean I believe Governance Facilitators ought to have this power, but that this is the intent of of the author and majority of voting MKR.
+
+**12.1: Bootstrapping Governance Votes**
+
+Governance Facilitators are empowered to use broad judgment when exercising their ability to make direct Atlas interpretations and edits to the Atlas and the Scope Artifacts, when this relates to unintended consequences or mistakes in the affected documents, *as well as when this relates to time-sensitive issues with the Launch Project.*
```

Comment:
Not the best way of phrasing this. Maybe:
```suggestion
Governance Facilitators are empowered to use broad judgment when exercising their ability to make direct Atlas interpretations and edits to the Atlas and the Scope Artifacts, when this relates to
* Unintended consequences in the affected documents
* Mistakes in the affected documents
* Time-sensitive issues with the Launch Project.
```

Or somesuch. Probably should use the numbering system.

Reply:
Relating to time-sensitive issues with the Launch Project is a very broad brush.